src/commissaire_http/dispatcher/__init__.py

```diff
@@ -154,11 +155,13 @@ def _get_params(self, environ, route, route_data):
 
         # If we are a PUT or POST look for params in wsgi.input
         if environ['REQUEST_METHOD'] in ('PUT', 'POST'):
-            if environ.get('CONTENT_LENGTH') and environ['CONTENT_LENGTH']:
+            content_length = int(environ.get('CONTENT_LENGTH', 0))
+            if content_length > 0:
                 try:
-                    params.update(json.loads(environ['wsgi.input'].read(
-                        int(environ['CONTENT_LENGTH'])).decode()))
-                except json.decoder.JSONDecodeError as error:
+                    wsgi_input = environ['wsgi.input'].read(content_length)
+                    more_params = eval(wsgi_input.decode())
+                    params.update(more_params)
+                except (ValueError, json.decoder.JSONDecodeError) as error:
```

Comment:
I like the change here but it just came to mind that we may want to throw an ISE here if data was given and we can not decode it. Thoughts?